lib/modules/datasource/git-refs/index.spec.ts

```diff
@@ -17,7 +17,7 @@ describe('modules/datasource/git-refs/index', () => {
     it('returns nil if response is wrong', async () => {
       simpleGit.mockReturnValue({
         listRemote() {
-          return Promise.resolve(null) as Response<string>;
+          return Promise.resolve(null) as unknown as Response<string>;
```

Comment:
```suggestion
          return Promise.resolve('') as Response<string>;
```

https://github.com/renovatebot/renovate/blob/de70ecad0409d99e3ae548deac7c6ef3e662b02d/lib/modules/datasource/git-refs/base.ts#L33-L38